### PR TITLE
This is no longer a valid link...

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

File: docs/review/usage/healing.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
     Code healing is an early feature and might not work for every use case. We would love to hear feedback and suggestions from you as we further improve this feature.
 
 !!! warning
-    This feature is only available with the standard license. If you do not own such a license, please [buy one here](https://retest.de/pricing/).
+    This feature is only available with the standard license. If you do not own such a license, please [buy one here](https://account.retest.de/customer/billing-data).
```
Comment (diba1013) [inline docs/review/usage/healing.md:7]: @martin-v What is the best alternative link for that? I would suggest https://account.retest.de/subscriptions. I never completed this setup...

Comment (martin-v) [reply docs/review/usage/healing.md:7]:
```suggestion
    This feature is only available with the standard license. If you do not own such a license, please write a mail to sales@retest.de.
```

File: docs/review/usage/healing.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ When you accept breaking changes, they are saved until you press "Apply changes"
 
 The following conditions will not trigger code healing:
 
-1. Your license does not suffice. In this case you need to [upgrade](https://retest.de/pricing/) your license.
+1. Your license does not suffice. In this case you need to [upgrade](https://account.retest.de/customer/billing-data) your license.
```
Comment (diba1013) [inline docs/review/usage/healing.md:21]: There is a second link here 😄

Comment (roesslerj) [reply docs/review/usage/healing.md]: Good catch!

**Pull request merged**